sql/current_date_and_subquery_with_between_are_used.sql

```diff
@@ -0,0 +1,10 @@
+select
+  current_date,
+  column_a
+from
+  dataset.table
+where
+  _table_suffix between '2022-06-01'
+  and (
+    select dt from dates
+  )
```

Comment:
Cloud you please add new line at the EOF?

Reply:
Thanks!
I fixed it in this commit
https://github.com/hirosassa/bqvalid/pull/16/commits/620010661546f167d617d935607add55805bc304